builtin/mainmenu/content/dlg_contentdb.lua

```diff
@@ -345,7 +345,9 @@ local function get_formspec(dlgdata)
 					core.colorize(mt_color_green, package.title) ..
 							core.colorize("#BFBFBF", " by " .. package.author)), "]",
 
-			"textarea[", img_w + 0.25, ",0.75;", cell_w - img_w - 0.25, ",", cell_h - 0.75, ";;;",
+			"label[", img_w + 0.25, ",0.75;", cell_w - img_w - 0.25, ",", cell_h - 0.75, ";",
+				core.formspec_escape(package.short_description), "]",
+			"tooltip[", img_w + 0.25, ",0.75;", cell_w - img_w - 0.25, ",", cell_h - 0.75, ";",
```

Comment:
The tooltip doesn't help with making the description readable since there is no wrapping in the tooltip. It's all shown in one long line that goes off-screen. You could use `core.wrap_text` for a simple fix.